src/main/java/com/sergiofah/model/Product.java

```diff
@@ -1,11 +1,17 @@
 package com.sergiofah.model;
 
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
 import javax.persistence.*;
-import java.io.Serializable;
 
+@Data
```

Comment:
`@Data` have`@EqualsAndHashcode` and `@ToString` annotation inside. These both  annotations decrease the query performance when consuming a database therefore it's necessary to substitute for `@Getter` and `@Setter`. Please, notice that this only applies to Entity classes.